test/framework/fileConversion.go

```diff
@@ -112,17 +157,29 @@ func doCmd(osCmd string, osArgs ...string) error {
 	return nil
 }
 
-// copyIfNotPresent checks for the src file in the tgtDir.  If it is not there, it attempts to copy if from src to tgtdir.
+// copyIfNotPresent checks for the src file in the tgtDir.  If it is not there, it attempts to copy it from src to tgtdir.
 // If a copy is performed, the path to the copy is returned.
 // If no copy is performed, the original src string is returned.
 func copyIfNotPresent(src, tgtDir string) (string, error) {
-	base := filepath.Base(src)
-	// Only copy the source image if it does not exist in the temp directory
-	if _, err := os.Stat(filepath.Join(tgtDir, base)); err != nil {
-		if err := doCmd("cp", "-f", src, tgtDir); err != nil {
-			return "", err
-		}
-		src = filepath.Join(tgtDir, base)
+	ret := filepath.Join(tgtDir, filepath.Base(src))
+	_, err := os.Stat(ret)
+	if err != nil && !os.IsNotExist(err) {
+		return "", errors.Wrap(err, "Unexpected error stating file")
+	}
```

Comment:
at this point err == nil or == IsNotExist. If err == nil we don't want to do the cp.